### DISCUSSION TITLE
Renaming Priorities

### DISCUSSION THREAD

**Discussion by tina-hague** [Patribots4738/Crescendo2024#118] [Q&A]
Our priority naming is a little funky. Since we are a team we should all decide on what the deadline/priority labels should be & agree on them. Saying 2 weeks, tomorrow, or 3 days is a little confusing since it does not update depending on the time that has passed but if the team likes that more we can use it. We also can use P1 - P5 as long as we all know what order it goes in. We can also just use random words or emojis like we have now but they are currently somewhat bland & we want input!!! Idk how this discussion works so lmma say **you can comment your suggestion & people can reply with how they feel (ie checkmark or big bad x).** Democracy! 😀

**Comment by GalexY727** [comment]
Democracy! 😀

**Comment by GalexY727** [comment]
OK hear me out:
😴
😇
😮
😥
😣

**Reply by GalexY727** [reply]
I forgot the cherry on top:
😡

**Comment by jacueblol** [comment]
Well I always liked P0-P4, but I do agree I don't like how it doesn't update.

**Reply by GalexY727** [reply]
I think if we went numbered it would be better if we dropped the P and just did 0-5

**Reply by GalexY727** [reply]
I don't like 0-4 because there isn't enough wiggle room. 0 would mean whenever and 4 would mean NOW, but what if I wanted something done "soon 😏"... it just doesn't click with my head enough to have only two options that arent the extrema

**Reply by GalexY727** [reply]
Having a 0-5 or 6 might make it easier to pick and choose if we like accuracy, but the saying is "Keep it simple stupid" not "add all the details uwu"

**Reply by tina-hague** [reply]
We can do a mix of both? Your brains process it differently so surely other people do to

**Reply by tina-hague** [reply]
Having more than just 😎 or 5 or just a word would be beneficial that way if someone doesn't fully understand the 😎 or the 5 or the "we chillin" then the other ones will compensate for it

**Reply by jacueblol** [reply]
Do what ur saying is that we have like...

- 0 : we chillin 😎 
- 1 : ooo something to do 👀 
- 2 : let's get a move on 🏃‍♂️ 
- 3 : go!!! please 🥺
- 4 : ok now let's really really get this done 😨 
- 5 : panicking 😨 😠 😮‍💨 😵‍💫 🤒 😨 
- 6 : bad news bear 🧸

**Reply by tina-hague** [reply]
Yes! Bears also yes!!! 🧸

**Reply by jacueblol** [reply]
@GalexY727 what do you think?

**Reply by GalexY727** [reply]
I love the bear, I think that if we were to remove 4 and 5 it might-haps be better because we don't want to make the standard be to get behind schedule and use those priorities

**Reply by GalexY727** [reply]
also a good news geese 🦢 instead of we chillin' of course

**Reply by tina-hague** [reply]
I like these very much! :D I think they'll make sense to most people but we can still talk about it tomorrow during scrum or just randomly

**Reply by Oliver-Cushman** [reply]
i like good news geese

**Reply by Hadizah454** [reply]
I like the bad news bears one🐻

**Comment by tina-hague** [comment]
![image](https://github.com/Patribots4738/Crescendo2024/assets/148731136/85ea9c1f-5522-451c-8c07-c55a6316a54f)
?

**Reply by tina-hague** [reply]
I FORGOT THE NUMBERS########
![image](https://github.com/Patribots4738/Crescendo2024/assets/148731136/9b62af67-fcf2-4d70-81dd-3d39c6964c40)

**Comment by jacueblol** [comment]
This was resolved!! GOOD NEWS GEESE
